From 3040e56a73d533b0fbcd452dd81d8c87e4810ed6 Mon Sep 17 00:00:00 2001
From: RS <132844576+RuimingShen@users.noreply.github.com>
Date: Wed, 12 Feb 2025 10:53:24 -0800
Subject: [PATCH 014/150] Update Dockerfile.cuda11.8

fix problem with last line, for more info regarding this problem see this link: https://stackoverflow.com/questions/61055324/docker-cannot-execute-binary-file add slack channel software fix in this file at line 63, update sdk version 4.0->4.1
---
 setup/Dockerfile.cuda11.8 | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/setup/Dockerfile.cuda11.8 b/setup/Dockerfile.cuda11.8
index 68753380..6e5b3750 100644
--- a/setup/Dockerfile.cuda11.8
+++ b/setup/Dockerfile.cuda11.8
@@ -19,7 +19,7 @@ RUN ln -fs /usr/share/zoneinfo/$TZ /etc/localtime \
     && rm -rf /var/lib/apt/lists/*
 
 # install Zed SDK
-RUN wget https://download.stereolabs.com/zedsdk/4.0/cu118/ubuntu20 -O zed_sdk.run
+RUN wget https://download.stereolabs.com/zedsdk/4.1/cu118/ubuntu20 -O zed_sdk.run
 RUN chmod +x zed_sdk.run
 RUN ./zed_sdk.run -- silent
 
@@ -60,7 +60,7 @@ RUN cd /catkin_ws/src && git clone https://github.com/ros-perception/radar_msgs.
    && cd radar_msgs && git checkout noetic 
 
 RUN source /opt/ros/noetic/setup.bash && cd /catkin_ws && rosdep install --from-paths src --ignore-src -r -y
-RUN source /opt/ros/noetic/setup.bash && cd /catkin_ws && catkin_make -DCMAKE_BUILD_TYPE=Release
+RUN source /opt/ros/noetic/setup.bash && cd /catkin_ws && catkin_make -DCMAKE_BUILD_TYPE=Release -j1
 
 # Copy requirements.txt from host (now relative to parent directory)
 COPY requirements.txt /tmp/requirements.txt
@@ -80,4 +80,4 @@ RUN echo "source /catkin_ws/devel/setup.bash" >> /home/${USER}/.bashrc
 # BASE END CONFIG
 WORKDIR /home/${USER}
 
-ENTRYPOINT ["/bin/bash"]
\ No newline at end of file
+ENTRYPOINT [ "/bin/bash", "-l", "-c" ]
-- 
2.38.1

